src/ng/interval.js

```diff
@@ -4,10 +4,18 @@ var $intervalMinErr = minErr('$interval');
 
 /** @this */
 function $IntervalProvider() {
-  this.$get = ['$rootScope', '$window', '$q', '$$q', '$browser',
-       function($rootScope,   $window,   $q,   $$q,   $browser) {
+  this.$get = ['$$intervalFactory', '$window',
```

Comment:
I assume we don't care that this is "breaking" the decoration of $interval by including a private service?

Reply:
Why does it break it?

Reply:
Now that you ask ... doesn't look like it breaks anything (and it relied on a private service before)